lib/utils.pm

```diff
@@ -0,0 +1,21 @@
+package utils;
+
+use base Exporter;
+use Exporter;
```

Comment:
No, use base is require + push(@ISA) - but this is just a copy of the one in os-autoinst-distri-opensuse